cdap-api/src/main/java/co/cask/cdap/api/mapreduce/MapReduceContext.java

```diff
@@ -47,6 +51,14 @@
 
   /**
    * Overrides the input configuration of this MapReduce job to use
```

Comment:
And that's why we needed to make the change! 'Till now, we didn't have a way to distinguish...